articles/libraries/_includes/_embedded_sso.md

```diff
@@ -0,0 +1,5 @@
+### SSO with embedded authentication
+
+If you use embedded login, then you can only implement SSO if your applications are [first-party clients](/clients/client-types#first-party-client) who share the same top level domain (for example, `app1.acme.com` and `app2.acme.com`). 
```

Comment:
@sjudson can you 👍 or 👎 this message? I want to make sure we're communicating this correctly.